packages/semantic-ui/package.json

```diff
@@ -27,8 +27,8 @@
     ]
   },
   "peerDependencies": {
-    "@rjsf/core": "^5.0.0-beta.1",
-    "@rjsf/utils": "^5.0.0-beta.1",
+    "@rjsf/core": "^5.0.0-beta.2",
+    "@rjsf/utils": "^5.0.0-beta.2",
```

Comment:
Jacques, I don't believe you need to change these since they have the `^` on them.